packages/constants/package.json

```diff
@@ -19,5 +19,10 @@
   },
   "keywords": [],
   "author": "",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "@types/mocha": "^5.2.7",
```

Comment:
We might want to use exact versions to be consistent?

Reply:
Oh, actually I think we do not need this dependency. will remove it.

Reply:
removed